### ISSUE TITLE
'RecursionError: maximum recursion depth exceeded in comparison' after 700+ invocations of @time_trigger("cron(...)") function

### ISSUE THREAD

**Issue by huonw** [custom-components/pyscript#132]
I have a function that runs every 5 minutes, defined like:

```python
UPDATE_MINUTES = 5

@time_trigger(f"cron(*/{UPDATE_MINUTES} * * * *)")
@service
def calculate_evaporation():
    ...
```

After running for ~2.5 days (3704.5 minutes, i.e. 740 invocations), every invocation (both automated cron and manual service calls) starts to fail with the following error:

```
2021-01-01 15:35:00 ERROR (MainThread) [custom_components.pyscript.file.evaporation.calculate_evaporation] Exception in <file.evaporation.calculate_evaporation> line 146:
        state.set(rate_var, round(rate, 2))
                                        ^
RecursionError: maximum recursion depth exceeded in comparison
```

PyScript version: 1.0.0 (eb4dde9c72c5bb25533820082ad69660495389c7)

**Comment by craigbarratt** [comment]
This was fixed in 1.1.0 with commit 2298017; see issue #87.

**Comment by huonw** [comment]
Ah, great, apologies and thanks!
